runtime/centrifuge/src/xcm.rs

```diff
@@ -203,11 +204,13 @@ pub struct CurrencyIdConvert;
 /// handle it on their side.
 impl Convert<CurrencyId, Option<MultiLocation>> for CurrencyIdConvert {
 	fn convert(id: CurrencyId) -> Option<MultiLocation> {
-		match id {
-			CurrencyId::Tranche(_, _) => None,
-			_ => OrmlAssetRegistry::multilocation(&id).ok()?,
-			// todo(nuno): verify this will work correctly
-		}
+		OrmlAssetRegistry::metadata(id)
+			.filter(|m| {
+				m.additional
+					.transferability
+					.includes(AssetTransferability::Xcm)
+			})
+			.and_then(|_| OrmlAssetRegistry::multilocation(&id).ok()?)
```

Comment:
We can just write the conversion from `m.location: Option<VersionedMultiLocation` to `MultiLocation` here instead of calling `OrmlAssetRegistry::multilocation` which looks up the `location` field we already have and does said conversion

Reply:
Done